server/documents/modules/dropdown.html.eco

```diff
@@ -2889,16 +2889,16 @@ themes      : ['Default', 'GitHub', 'Material']
           <td>Hides dropdown</td>
         </tr>
         <tr>
-          <td>clear</td>
-          <td>Clears dropdown of selection</td>
+          <td>clear (preventChangeTrigger)</td>
+          <td>Clears dropdown of selection. Set <code>preventChangeTrigger</code> to <code>true</code> to omit the change event</td>
```

Comment:
I think documenting default values for each new parameter helps users.

Reply:
@exoego Which parameters are you referring to? The only parameter is explained here 🤔

Reply:
sorry, did not recognized your suggestion... 🙄 😄